src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -211,6 +213,28 @@ private void validateSchema(final EventTypeBase eventType) throws InvalidEventTy
         }
     }
 
+    /**
+     * bugifx ARUHA-563
+     *
+     * @param eventTypeSchema
+     * @throws InvalidEventTypeException
+     */
+    private void checkSchemaValidity(final String eventTypeSchema) throws InvalidEventTypeException {
```

Comment:
I we really want to do it like that I suggest not to do that at all :(
I think the proper solution would be to find a proper validation library or to create a node.js microservice that will do that.